arm-service-map/2015-11-01-preview/swagger/arm-service-map.json

```diff
@@ -0,0 +1,2495 @@
+        }
+      ],
+      "description": "Base for all summaries."
+    },
+    "MachinesSummary": {
+      "description": "A summary of the machines in the workspace.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Summary"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/MachinesSummaryProperties"
+        }
+      }
+    },
+    "Relationship": {
+      "description": "A typed relationship between two entities.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ]
+    },
+    "Connection": {
+      "description": "A network connection.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Relationship"
+        }
+      ],
+      "x-ms-discriminator-value": "rel:connection",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ConnectionProperties"
+        }
+      }
+    },
+    "Acceptor": {
+      "description": "A process accepting on a port.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Relationship"
+        }
+      ],
+      "x-ms-discriminator-value": "rel:acceptor",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RelationshipProperties"
+        }
+      }
+    },
+    "MachineCollection": {
+      "type": "object",
+      "description": "Collection of Machine resources",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Machine"
+          },
+          "description": "collection of Machine Resources"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "the URL to the next set of resources"
+        }
+      }
+    },
+    "ProcessCollection": {
+      "type": "object",
+      "description": "Collection of Process resources",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Process"
+          },
+          "description": "collection of Process Resources"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "the URL to the next set of resources"
+        }
+      }
+    },
+    "PortCollection": {
+      "type": "object",
+      "description": "Collection of Port resources",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Port"
+          },
+          "description": "collection of Port resources"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "the URL to the next set of resources"
+        }
+      }
+    },
+    "ClientGroupMembersCollection": {
```

Comment:
Ah. Found what probably should have been called `ClientGroupMemberCollection` -- this isn't referenced, either drop the `s` here or fix up the reference to it.